File: Week_0/yobinmok/2562.py
n = []
for i in range(9):
    n.append(int(input()))

max = 0 # max함수 이용해도 됨
for i in range(9):
    if n[i] > max:
        max = n[i]
        index = i + 1

print(max)
print(index) # a.index(max) + 1

File: Week_0/SeJun/2562.cpp
#include <iostream>

using namespace std;

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);
	int num, index;
	int max = 0;
	for (int i = 0; i < 9; i++) {
		cin >> num;
		if (num > max) {
			max = num;
			index = i;
		}
	}
	cout << max << "\n";
	cout << index + 1;
}